app/views/listings/index.html.erb

```diff
@@ -13,16 +13,15 @@
       <% @listings.each do |listing| %>
         <li class="listing-slide" id="listing-slide">
           <%= link_to user_listing_path(listing.user.slug, listing) do %>
-            <% if listing.listing_images.any? %>
-              <%= image_tag listing.listing_images.first.image.url %>
-            <% end %>
-            <span class="d-hover"><br>
-              <%#= link_to listing.title, user_listing_path(listing.user_id, listing) %><br>
-              Property Category: <%= listing.category.name %><br>
-              Property Description: <%= listing.description %><br>
-              Nightly Rate: <%= number_to_currency(listing.currency) %><br>
-              Max Guests: <%= listing.max_guests %><br>
-            </span></a></li>
+          <% if listing.listing_images.any? %>
```

Comment:
I agree to delete the commented out code. Will we need to add a link in here still tho?